### ISSUE TITLE
Issue when foldernames have spaces 

### ISSUE THREAD

**Issue by treborxy** [lrademacher/PDFTag#19]
Maybe I'm overlooking something, but I had problems with foldernames containing spaces. PT just display nothing when I choose such a folder...

**Comment by lrademacher** [comment]
Yeah, could be. Typically i am avoiding such directory names. You could have a look at it, if you want to have it fixed quickly. Otherwise i will see when i find time to work on PDFTag and improve this.
